### ISSUE TITLE
Deepface.analyze doesn't show Gender Logits

### ISSUE THREAD

**Issue by liechtym** [serengil/deepface#593]
Awesome library! Thanks for all your work on this.

I'm having a small problem with gender outputs on DeepFace.analyze()

My Code:
```
demography = DeepFace.analyze(img_path = file_prefix + 'image1.jpeg', actions = ['gender'])
```

My Output:
```
{'gender': 'Man', 'region': {'x': 137, 'y': 79, 'w': 291, 'h': 291}}
```

The output example given in the code is a bit different from my output.
https://github.com/serengil/deepface/blob/master/deepface/DeepFace.py#L294-L321 

From what I understand, the gender logits for each class should be returned in the property "gender" along with the "dominant gender" property that assigns a specific label. Right now all that's being returned is the "gender" property and it only provides the dominant gender prediction. 

This is relevant for me because I'm trying to test if the model is good at identifying the ambiguity in gender among non-binary individuals. This would only be possible if I am able to access the gender class logits.

Thanks for you help!
Matthew

PS: I installed using `pip install deepface`

UPDATE: 

I see the gender logits were added in this commit:
https://github.com/serengil/deepface/commit/3f29c6a606ca5fc0454595521c7c88ccb5aebbfe

So it looks like this problem would be fixed by publishing a new version in pip.

**Comment by serengil** [comment]
yes this is not published yet

meanwhile you can download the source code and run it.
